src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -593,7 +593,9 @@ public static PSDataCollection<PSObject> GetPSDataCollection(object inputValue)
         /// <param name="second">Object to compare first to.</param>
```

Comment:
@daxian-dbw, your last commit had 2 failures in `PowerShell-CI-macos`
Built-in type information tests.Should have correct number of built-in type items in type table
```powershell
Expected exactly 273, but got 271.
at <ScriptBlock>, /Users/runner/runners/2.163.1/work/1/s/test/powershell/engine/ETS/TypeTable.Tests.ps1: line 22
22:         $types.Count | Should -BeExactly ($isUnixStatEnabled ? 272 : 273)
```
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Diagnostics.TraceSource.dll; SideIndicator=<=}, @{InputObject=System.Runtime.Serialization.Primitives.dll; SideIndicator=<=}).
at <ScriptBlock>, /Users/runner/runners/2.163.1/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```